docs/demos/UseCases.story.tsx

```diff
@@ -86,6 +86,7 @@ export const CyberSecurity = () => {
       theme={darkTheme}
       draggable
       layoutType="forceDirected2d"
+      edgeShape="curve"
```

Comment:
I don't think `shape` is a good word for this. What do you think about `edgeInterpolation: 'linear' | 'curved';`

I checked out what cytoscape calls this and don't really love it either.

> `curve-style` : The curving method used to separate two or more edges between two nodes ([demo](https://js.cytoscape.org/demos/edge-types)); may be [haystack](https://js.cytoscape.org/#style/haystack-edges) (default, very fast, bundled [straight](https://js.cytoscape.org/#style/straight-edges) edges for which loops and compounds are unsupported), straight (straight edges with all arrows supported), [straight-triangle](https://js.cytoscape.org/#style/straight-triangle-edges) (straight triangle edges), [bezier](https://js.cytoscape.org/#style/bezier-edges) (bundled curved edges), [unbundled-bezier](https://js.cytoscape.org/#style/unbundled-bezier-edges) (curved edges for use with manual control points), [segments](https://js.cytoscape.org/#style/segments-edges) (a series of straight lines), [taxi](https://js.cytoscape.org/#style/taxi-edges) (right-angled lines, hierarchically bundled). Note that haystack edges work best with ellipse, rectangle, or similar nodes. Smaller node shapes, like triangle, will not be as aesthetically pleasing. Also note that edge endpoint arrows are unsupported for haystack edges.